app/controllers/brexit_checker_controller.rb

```diff
@@ -33,6 +43,39 @@ def show
   end
 
   def results
+    if accounts_enabled? && logged_in?
+      begin
+        user_data = Array(
+          update_session_tokens(
+            oidc.get_checker_attribute(
+              access_token: session[:access_token],
+              refresh_token: session[:refresh_token],
+            ),
+          ),
+        )
+
+        if user_data.empty? || criteria_keys != user_data.last["criteria_keys"]
+          user_data << {
+            check_id: SecureRandom.uuid,
+            timestamp: Time.zone.now,
+            criteria_keys: criteria_keys,
+          }
+
+          update_session_tokens(
+            oidc.update_checker_attribute(
+              access_token: session[:access_token],
+              refresh_token: session[:refresh_token],
+              data: user_data,
+            ),
+          )
+        end
+      rescue OidcClient::OAuthFailure
+        # this means the refresh token has been revoked or the
+        # accounts services are down
+        logout!
+      end
+    end
+
```

Comment:
This can be removed, it's been superseded by the JWT flow in #2207

Reply:
Ah, yes, of course.  I've now removed this code and the results no longer get saved to the account automatically.